.github/workflows/build-and-push-docker-image.yml

```diff
@@ -27,8 +27,19 @@ jobs:
         env:
           ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
         run: |
-          docker build . -t $ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA
+          image_id=$(docker build . -q -t $ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA)
+          echo "image_id=${image_id}" >> $GITHUB_ENV
           echo "::set-output name=image::$ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA"
           docker image tag $ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA $ECR_REGISTRY/$ECR_REPOSITORY:latest
           docker push $ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA
           docker push $ECR_REGISTRY/$ECR_REPOSITORY:latest
+      - name: Copy wasm artifact from the image
+        run: |
+          image_id="${{ env.image_id }}"
+          container_id=$(docker create $image_id)
+          docker cp "${container_id}:/home/cere/node-runtime-artifacts/node_runtime.compact.wasm" ./
+      - name: Upload wasm artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: node_runtime.compact.wasm
```

Comment:
Let's add a unique identifier to the artifact filename. Use GITHUB_SHA

Reply:
Added. Here is an [example](https://github.com/Cerebellum-Network/pos-network-node/actions/runs/2795433038)